test/txLib.js

```diff
@@ -11,12 +11,17 @@ import BN from 'bn.js';
 import { BigInt, equal } from 'jsbi';
 import { Period, Block, Tx, Input, Output, Outpoint, Type } from 'leap-core';
 import chai from 'chai';
+const ethers = require('ethers');
```

Comment:
why ethers?

Reply:
to be able to read structs from contract responses